docs/agents/sematext-agent/ansible.md

```diff
@@ -0,0 +1,129 @@
+
+## Setup Inventory
+An Ansible [inventory](https://docs.ansible.com/ansible/latest/inventory_guide/index.html) is a list of all hosts that will be managed by Ansible. They can be logically grouped by webservers, databases, production, etc. Check out the [official documentation](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#inventory-basics-formats-hosts-and-groups
+) to learn more.
+
+The inventory file can be saved anywhere, and in its simplest form for two hosts, it appears as shown below:
+
+```
+[servers]
+server1 ansible_host=15.15.225.101 ansible_port=22 ansible_ssh_user=ubuntu ansible_ssh_private_key_file=/home/ubuntu/server1-key.pem
+server2 ansible_host=15.15.225.102 ansible_port=22 ansible_ssh_user=ubuntu ansible_ssh_private_key_file=/home/ubuntu/server2-key.pem
+```
+
+where:
+
+- `server1`, `server2` are [aliases](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#inventory-aliases)
+- ansible_host is the IP address of each host
+- ansible_port is the SSH host port
+- ansible_ssh_user is the SSH username
+- ansible_ssh_private_key_file is the private SSH certificate file of each host
+
+If your inventory file is named `hosts` then you can verify whether you can ping all your hosts with the following command:
+
+```ansible -i hosts all -m ping```
+
+If the hosts are reachable, you should expect a response like the one below:
+
+```
+server1 | SUCCESS => {
+    "ansible_facts": {
+        "discovered_interpreter_python": "/usr/bin/python3"
+    },
+    "changed": false,
+    "ping": "pong"
+}
+server2 | SUCCESS => {
+    "ansible_facts": {
+        "discovered_interpreter_python": "/usr/bin/python3"
+    },
+    "changed": false,
+    "ping": "pong"
+}
+```
+
+
+## Sematext Agent Ansible Role
+The Sematext Agent role contains everything required to install Sematext Agent in a specified list of hosts according to the Ansible playbook that will be used. It can be installed either through Ansible Galaxy or manually from the [ansible-install-sematext repository](https://github.com/sematext/ansible-install-sematext).
+
+
+### Install from Ansible Galaxy
+Install the [Sematext Agent role from Ansible Galaxy](https://galaxy.ansible.com/ui/standalone/roles/sematext/spm-monitor-install/) on your control node by issuying the following command:
+
+```ansible-galaxy role install sematext.spm-monitor-install```
+
+
+### Manual Installation
+If you want to manually install the role, clone the [ansible-install-sematext](https://github.com/sematext/ansible-install-sematext) repo in the control node and copy this repo to `~/.ansible/roles/sematext.spm-monitor-install`.
+
+
+### Supported Platforms
+Linux distributions:
+
+- Ubuntu
+    - xenial
+    - bionic
+    - focal
+    - jammy
+- Debian
+    - wheezy
+    - jessie
+    - stretch
+    - buster
+    - bullseye
+    - bookworm
+- Red Hat Enterprise Linux 6, 7, 8, 9
+- Fedora 38
```

Comment:
Only 38?

Reply:
I don't believe 38 only, but I haven't tested older versions. v37 is already EOL. I can take few of those (e.g. v28 and v32) and see the result if you want?

Reply:
If it's EOL there is no point adding this. Plus I believe we should do the same for Ubuntu